includes/class-sensei-course.php

```diff
@@ -3886,6 +3889,28 @@ public function has_sensei_blocks( $course = null ) {
 
 		return false;
 	}
+
+	/**
+	 * Alters the redirect url after a user enrols to a course.
+	 *
+	 * @param String  $url  Default redirect url.
+	 * @param WP_Post $post Post object for course.
+	 *
+	 * @return String
+	 */
+	public static function alter_redirect_url_after_enrolment( $url, $post ) {
+
+		$course_id = $post->ID;
+		if ( Sensei_Course_Theme::has_sensei_theme_enabled( $course_id ) ) {
```

Comment:
I think @yscik is renaming this class to `Senseio_Course_Theme_Option` in a separate PR, I might need to refactor this to adapt to the new name.